src/components/FilterContainer.vue

```diff
@@ -102,7 +102,7 @@ export default {
       return this.canEdit && !this.doCollapse
     },
     listOfVisibleFilters () {
-      return this.filters.filter(filter => this.filtersToShow.includes(filter.name))
+      return this.filtersToShow.map(id => this.filters.find(filter => filter.name === id))
```

Comment:
Works is using the position in de array a good way to set the order of the filters ? maybe use a order prop ?